server/controllers/images.ts

```diff
@@ -0,0 +1,145 @@
+import path from 'path';
+
+import { ManagedUpload } from 'aws-sdk/clients/s3';
+import { encode } from 'blurhash';
+import config from 'config';
+import { kebabCase } from 'lodash';
+import sharp from 'sharp';
+import { v1 as uuid } from 'uuid';
```

Comment:
future v1 values can be predicted based on known past values, so this could be a v4?